shone/chemistry/translate.py

```diff
@@ -62,7 +62,7 @@ def isotopologue_to_mass(isotopologue):
     return (mass if mass != 0 else getattr(elements, isotopologue).mass) * u.u
 
 
-def species_name_to_fastchem_name(species, return_mass=False):
+def species_name_to_fastchem_name(species, charge, return_mass=False):
```

Comment:
We shouldn't require a charge, since some species don't have one.
```suggestion
def species_name_to_fastchem_name(species, charge=None, return_mass=False):
```

Reply:
This required argument is causing the tests to fail: https://github.com/bmorris3/shone/actions/runs/9303587794/job/25608838527?pr=13#step:10:293